model/Core/Vocabularies/RelationshipType.md

```diff
@@ -48,6 +48,7 @@ name completes the sentence:
 - hasAssessmentFor: Relates a `from` Vulnerability and each `to` Element with a security assessment. To be used with `VulnAssessmentRelationship` types.
 - hasAssociatedVulnerability: Used to associate a `from` Artifact with each `to` Vulnerability.
 - hasConcludedLicense: The `from` SoftwareArtifact is concluded by the SPDX data creator to be governed by each `to` license.
+- hasContactPoint: The `from` Artifact has a contact point `to` an Agent. The use of `hasContactPoint` type is constrained to `ContactPointRelationship` typed relationships. The type of contact (i.e. security) may be specified using a `ContactPointRelationship` element.
```

Comment:
```suggestion
- hasContactPoint: The `from` Artifact has each `to` Agent as a contact point. The use of `hasContactPoint` type is constrained to `ContactPointRelationship` typed relationships. The type of contact (i.e. security) may be specified using a `ContactPointRelationship` element.
```

Borrows the structure from `hasDataFile`, `hasInput`, and `hasOutput`.

Or alternatively, borrowing `hasEvidence`, `hasExample`, `hasSpecificition`, `hasTest`:

> Every `to` Agent is a contact point for the `from` Artifact.

(In any case, `to` here is an Agent but the current text looks like it collapsed the preposition "to" with the Agent `to`.)